### PR TITLE
linted and added polygon test

### PR DESCRIPTION
**Pull request by shershah010**
Linted and added polygon test.

### CODE REVIEW

File: harvest/api/polygon.py [hunk 1]
```diff
@@ -27,25 +27,16 @@ def __init__(self, path: str = None, is_basic_account: bool = False):
...
 
-    def setup(self, interval, trader_main=None):
-        self.watch_stock = []
-        self.watch_crypto = []
-
-        for sym in interval:
-            if is_crypto(sym):
-                self.watch_crypto.append(sym)
-            else:
-                self.watch_stock.append(sym)
```
Comment (shershah010) [inline harvest/api/polygon.py:38]: not used anymore

File: setup.py [hunk 2]
```diff
@@ -37,7 +37,7 @@ def finalize_options(self):
 
     def run(self):
         a = subprocess.run(
-            ["coverage", "run", "-m", "unittest", "discover", "-s", "tests"]
+            ["coverage", "run", "-m", "unittest", "discover", "-s", "tests/unittest"]
```
Comment (shershah010) [inline setup.py:40]: update path to tests run by github and `python setup.py test`

File: harvest/api/paper.py [hunk 3]
```diff
@@ -31,7 +31,7 @@ class PaperBroker(API):
         Interval.DAY_1,
     ]
 
-    def __init__(self, path: str = None, streamer=None):
+    def __init__(self, path: str = None, streamer=None, commission_fee=0):
```
Comment (shershah010) [inline harvest/api/paper.py:34]: Readded commission fee

File: harvest/api/paper.py [hunk 4]
```diff
@@ -159,10 +157,10 @@ def fetch_account(self) -> Dict[str, Any]:
         self.equity = self._calc_equity()
         self._save_account()
         return {
-            "equity": self.config["equity"],
-            "cash": self.config["cash"],
-            "buying_power": self.config["buying_power"],
-            "multiplier": self.config["multiplier"],
+            "equity": self.equity,
```
Comment (shershah010) [inline harvest/api/paper.py:160]: remove config since that is no longer accurate

File: tests/unittest/test_cli.py [hunk 5]
```diff
@@ -34,15 +34,15 @@ def test_bad_streamer(self):
             self.assertTrue(True)
 
     def test_start_basic(self):
-        crossover = os.path.dirname(os.path.realpath(__file__)) + "/../examples"
+        crossover = os.path.dirname(os.path.realpath(__file__)) + "/../../examples"
```
Comment (shershah010) [inline tests/unittest/test_cli.py:37]: update path

**Review by tfukaza — APPROVED** [review]

**Pull request merged**